examples/official-storybook/stories/addon-notes.stories.js

```diff
@@ -37,10 +26,20 @@ storiesOf('Addons|Notes', module)
     ))
   )
 ~~~
-    `)(() => (
-      <BaseButton label="Button with notes from inline github-flavored markdown - check the notes panel for details" />
-    ))
-  )
+    `;
+
+storiesOf('Addons|Notes', module)
```

Comment:
Please add these example to the other frameworks too

Reply:
Yep!